Description: debianize
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 lenstoggle (0.1-1) unstable; urgency=low
 .
   * source package automatically created by stdeb 0.6.0+git
Author: aking1012.com@gmail.com <aking1012.com@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ lenstoggle-0.1/LensToggle.egg-info/top_level.txt
@@ -0,0 +1 @@
+LensToggle
--- /dev/null
+++ lenstoggle-0.1/LensToggle.egg-info/requires.txt
@@ -0,0 +1 @@
+setuptools
\ No newline at end of file
--- /dev/null
+++ lenstoggle-0.1/LensToggle.egg-info/PKG-INFO
@@ -0,0 +1,16 @@
+Metadata-Version: 1.1
+Name: LensToggle
+Version: 0.1
+Summary: Simple App to Toggle Lenses.
+Home-page: http://www.github.com/aking1012/pyeggtut
+Author: aking1012.com@gmail.com
+Author-email: aking1012.com@gmail.com
+License: GPL
+Description: Simple App to Toggle Lenses.  I did not like that unity --replace & moved my screenlets around, so I took that part out.  Just log out and log in.  
+Keywords: python lens toggle Gtk
+Platform: UNKNOWN
+Classifier: License :: Not OSI Approved :: BSD + Non-Commercial
+Classifier: Programming Language :: Python
+Classifier: Development Status :: 3 - Alpha
+Classifier: Intended Audience :: Users that want to temporarily disable lenses without uninstalling them
+Classifier: Topic :: Ubuntu Lenses
--- /dev/null
+++ lenstoggle-0.1/LensToggle.egg-info/dependency_links.txt
@@ -0,0 +1 @@
+
--- /dev/null
+++ lenstoggle-0.1/LensToggle.egg-info/SOURCES.txt
@@ -0,0 +1,8 @@
+setup.py
+LensToggle/Toggler.py
+LensToggle/__init__.py
+LensToggle.egg-info/PKG-INFO
+LensToggle.egg-info/SOURCES.txt
+LensToggle.egg-info/dependency_links.txt
+LensToggle.egg-info/requires.txt
+LensToggle.egg-info/top_level.txt
\ No newline at end of file
